find_max.rb

```diff
@@ -0,0 +1,19 @@
+
+# Find the maximum element between index 0 and max_index, exclusive
+
+def find_max(array, max_index)
+  max = nil
+  if max_index == 0
+    puts "Array is empty"
```

Comment:
I think a return statement would be better than a puts, then the code wouldn't continue to run and would just return immediately.
ie: return "Array is empty"